### ISSUE TITLE
Lmao u just stole my code and didnt give credits

### ISSUE THREAD

**Issue by hoemotion** [Social404/SelfBot#1]
this code was stolen from my https://github.com/hoemoion/Karuma/ project

**Comment by iGetUploaded** [comment]
That's all he does is steal code... shouldn't be surprised at all

**Comment by pxladin** [comment]
You're brave for publicly stating that this absolutely garbage code is yours

**Comment by hoemotion** [comment]
> You're brave for publicly stating that this absolutely garbage code is yours

Garbage code?
Hahhahaahahha

**Comment by pxladin** [comment]
Just look at line 325 to 542, like what is that? Do you not know what loops are?

**Comment by hoemotion** [comment]
> Just look at line 325 to 542, like what is that? Do you not know what loops are?

Yeah i changed that to a loop, but i forgot to update that in github. Thx for the hint

**Comment by suuuub** [comment]
Your code is garbage bro

**Comment by FlyingndCoding** [comment]
No offense i think we shall do an github request to take down his github @iGetUploaded @hoemotion

**Comment by iGetUploaded** [comment]
He would just make a new one and keep doing it

**Comment by FlyingndCoding** [comment]
Github has ip ban too tbh and how much will u use proxy he just learnt how to blur videos,cheat and copy people codes and argue with them

**Comment by iGetUploaded** [comment]
He would prob change IP or some dumb shit

**Comment by hoemotion** [comment]
> Your code is garbage bro

Lol I made this code without even knowing how asyncio stuff shall be handled but bro please take a look at your own NukeBot before criticizing my code and once i have the time i will clean it up

**Comment by hoemotion** [comment]
> 

I already did but github staff didnt care, the only thing social404 added to the code was the token grabber (he removed it already but if you take a look at the commits you can still see it)

**Comment by hoemotion** [comment]
> No offense i think we shall do an github request to take down his github @iGetUploaded @hoemotion

I already did but github staff didnt care, the only thing social404 added to the code was the token grabber (he removed it already but if you take a look at the commits you can still see it)
